mantidproject/mantid
# User file load from batch

Merge after #22335

**Description of work**
When specified in a batch file a user file was not being loaded into the table in the new ISIS SANS GUI.
**To test:**


* Get the ISIS test data from downloads.mantidproject.org
* The sans relevant data is in the loqdemo folder.
* Replace the .csv file with the one below
[batch_mode_reduction.csv.zip](https://github.com/mantidproject/mantid/files/1971204/batch_mode_reduction.csv.zip)
* In mantid open the ISIS SANS V2 GUI.
* Select MaskFile.txt as the userfile and batch_mode_reduction.csv as the batch file.
* Confirm that the first row of the table contains MaskFile.txt in the User File column

Fixes #22374 . 

**Release Notes** 
Have updated release notes.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
